storage/src/test/java/com/tencent/rss/storage/handler/impl/MultiStorageHdfsClientReadHandlerTest.java

```diff
@@ -319,7 +319,7 @@ private void compareDataAndIndex(
     return data;
   }
 
-  private List<ByteBuffer> readData(MultiStorageHdfsClientReadHandler handler) throws IllegalStateException {
+  private List<ByteBuffer> readData(UploadedStorageHdfsClientReadHandler handler) throws IllegalStateException {
```

Comment:
At first glance, readData is very similar to readShuffleData.

readUploadedData or other ?

Reply:
It's a method belongs to a test class.